index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Jake's Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <div id="head_logo">Jake Felice</div>
+        <div id="menu-button"></div>
+    </header>
+    <section id="intro">
+        <div id="intro-content">
+            <h1>Hey There. I'm Jake</h1>
+            <h3>I am a Software Developer</h3>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-icons">
+            <a target="_black" href="#" class="social-icons" id="medium"></a>
+            <a target="_black" href="https://twitter.com/jasonfelice0" class="social-icons" id="twitter"></a>
+            <a target="_black" href="#" class="social-icons" id="logo"></a>
+            <a target="_black" href="#" class="social-icons" id="linkedin"></a>
+            <a target="_black" href="https://github.com/jasonfelice" class="social-icons" id="github"></a>
+        </div>

```

Comment:
- [ ] Please make sure the list of social icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag) cause it will be better semantically that way.